doc/source/serve/llm/quick-start.rst

```diff
@@ -354,6 +354,166 @@ Advanced Usage Patterns
+            :language: python
+            :start-after: __cross_node_pp_example_start__
+            :end-before: __cross_node_pp_example_end__
+
+    .. tab-item:: YAML
+        :sync: yaml
+
+        .. code-block:: yaml
+
+            # config.yaml
+            applications:
+            - args:
+                llm_configs:
+                  - model_loading_config:
+                      model_id: llama-3.1-8b
+                      model_source: meta-llama/Llama-3.1-8B-Instruct
+                    accelerator_type: L4
+                    deployment_config:
+                      autoscaling_config:
+                        min_replicas: 1
+                        max_replicas: 1
+                    engine_kwargs:
+                      pipeline_parallel_size: 2
+                      distributed_executor_backend: ray
+                      max_model_len: 8192
+              import_path: ray.serve.llm:build_openai_app
+              name: llm_app
+              route_prefix: "/"
+
+Combined tensor and pipeline parallelism
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For extremely large models, you can combine both tensor and pipeline parallelism. The total number of GPUs is the product of `tensor_parallel_size` and `pipeline_parallel_size`.
+
+The following example shows how to configure a model with both TP and PP (4 GPUs total):
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. literalinclude:: ../doc_code/cross_node_parallelism_example.py
+            :language: python
+            :start-after: __cross_node_tp_pp_example_start__
+            :end-before: __cross_node_tp_pp_example_end__
+
+    .. tab-item:: YAML
+        :sync: yaml
+
+        .. literalinclude:: ../doc_code/cross_node_parallelism_example.py
+            :language: yaml
+            :start-after: __yaml_cross_node_tp_pp_example_start__
+            :end-before: __yaml_cross_node_tp_pp_example_end__
+
+Custom placement groups
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can customize how Ray places vLLM engine workers across nodes through the `placement_group_config` parameter. This parameter accepts a dictionary with `bundles` (a list of resource dictionaries) and `strategy` (placement strategy).
+
+Ray supports the following :ref:`placement strategies <pgroup-strategy>`:
+
+- **PACK** (default): Packs all workers onto as few nodes as possible on a best-effort basis. If workers can't fit on a single node, they spill to other nodes. This enables cross-node placement when single-node resources are insufficient.
+- **SPREAD**: Spreads each worker onto separate nodes on a best-effort basis. If strict spreading isn't feasible, workers can be placed on overlapping nodes.
+- **STRICT_PACK**: All workers must be placed on a single node. The deployment fails if workers can't fit on one node.
+- **STRICT_SPREAD**: Each worker must be on a separate node. The deployment fails if this constraint can't be satisfied.
```

Comment:
Let's not explain these and create redundancy. Ray core reference explains these in details.